### PR TITLE
Complete inputs and IPC for module 1

### PR DESCRIPTION
**Pull request by chance-nelson**

### CODE REVIEW

File: src/js/module1.js [hunk 1]
```diff
@@ -13,30 +13,47 @@ const submit_button                = document.getElementById("submitButton");
...
+/**
+ * @brief Template for a nucleotide sequence
+ */
+class Sequence {
+    constructor(identifier, nucleotide_sequence, ranges) {
+        this.identifier = identifier;
+        this.nucleotide_sequence = nucleotide_sequence;
+        this.ranges = ranges;
+    }
+};
```
[the remark this answers is not in the document]
Comment (chance-nelson) [reply src/js/module1.js:48]: Keeping this

**Comment by turanoo** [comment]
screen shot 2019-02-20 at 10 21 35 am
@chance-nelson can you look into this?? 
I am getting two headers and range selector when I try to upload a fasta file

**Comment by chance-nelson** [comment]
> screen shot 2019-02-20 at 10 21 35 am
> 
> @chance-nelson can you look into this?? I am getting two headers and range selector when I try to upload a fasta file

@turanoo This is working as intended. That fasta file possesses multiple sequences, and the client wants the program to bulk-add them in that case.

**Comment by turanoo** [comment]
@chance-nelson Great! Thanks for the explanation.

**Comment by turanoo** [comment]
@chance-nelson The input box for Fasta sequences is not working for some reason. When I try to manually paste text it's not letting me. It is also not giving me the range when I try to just write the sequences. Is this normal?

**Comment by chance-nelson** [comment]
@turanoo I have added event listeners to the sequence identifier, and also an editable textarea for changing the nucleotide sequence. However, I cannot test changing the nucleotide sequence due to a scrolling problem.

@skate54345 can you handle fixing this?

**Comment by turanoo** [comment]
Will wait on @skate54345 to fix the scrolling issue. Everything else look good tho 👍

**Review by turanoo — APPROVED** [review]

**Pull request merged**